dropwizard-dependencies/pom.xml

```diff
@@ -477,6 +471,11 @@
                 <artifactId>mockito-junit-jupiter</artifactId>
                 <version>${mockito.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-inline</artifactId>
+                <version>${mockito.version}</version>
+            </dependency>
```

Comment:
Do we need this new dependency? I don't see any changes in the usage of Mockito in our tests.

Reply:
Unfortunately, `CloseableHttpResponse` is a `final` class now and we cannot mock this any more. To simplify the migration, I wanted to reuse the existing tests. `mockito-inline` allows to mock `final` classes, so the tests work again after adding this dependency.

Reply:
@zUniQueX Okay, got it. Thanks for the explanation!

Feel free to merge the PR. :smile:

Reply:
Thanks for the fast review. There probably is a more elegant way than using `mockito-inline`. But in this PR I just wanted to get the dependency upgrade done. So I'll leave this as-is for now.